src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.MultiSegment.cs

```diff
@@ -2379,8 +2379,9 @@ private bool ConsumeSingleLineCommentMultiSegment(ReadOnlySpan<byte> localBuffer
         Done:
             if (HasValueSequence)
             {
-                SequencePosition end = new SequencePosition(_currentPosition.GetObject(), _currentPosition.GetInteger() + idx);
+                SequencePosition end = new SequencePosition(_currentPosition.GetObject(), _currentPosition.GetInteger());
                 ReadOnlySequence<byte> commentSequence = _sequence.Slice(start, end);
+                commentSequence = commentSequence.Slice(2, commentSequence.Length - 2 - (expectLF ? 1 : 0));
```

Comment:
Can we try to avoid the double sequence slice? Is it possible to have `commentSequence` be set to the right value in the first slice, by setting start/end correctly?

Reply:
Working on this.